model/registry/process.yaml

```diff
@@ -76,3 +76,38 @@ groups:
           An additional description about the runtime of the process, for example
           a specific vendor customization of the runtime environment.
         examples: 'Eclipse OpenJ9 Eclipse OpenJ9 VM openj9-0.21.0'
+  - id: registry.process.attributes
+    prefix: process
+    type: attribute_group
+    brief: >
+        Metric attributes that appear on some process metrics.
+    attributes:
+      - id: cpu.state
+        brief: "The CPU state for this data point. A process SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
```

Comment:
Also here: It talks about metric things (data points) but we are in the registry. This should be adapted when you use the attribute `ref` in the process-metrics yaml file.

Reply:
Is it possible to add a brief with a ref? In that case I will just delete it here and add the briefs in with the refs in `process-metrics.yaml`

Reply:
Yeah, you can override when using `ref`. Like here for example: https://github.com/open-telemetry/semantic-conventions/blob/main/model/trace/http.yaml#L66

But if this is just used in metrics as you mentioned above, then leaving here is fine.